action.yml

```diff
@@ -29,11 +29,16 @@ inputs:
     description: Should Cygwin's bin directory be added to the system PATH?
     required: false
     default: true
+  check-hash:
+    description: Check the hash of the installer
+    required: false
+    default: 'true'
```

Comment:
This quoting is inconsistent

Reply:
> This quoting is inconsistent

The problem is that the value of `default` [must be a string](https://docs.github.com/en/actions/sharing-automations/creating-actions/metadata-syntax-for-github-actions#inputsinput_iddefault), whereas with `default: true` it would be a boolean.
I didn't touch the existing code (but we should fix it too)

Reply:
I've updated this PR with 813884572636f38db2489ac5ed8cba1ba72f2889
PS: Visual Studio Code suggested me that we shouldn't use a boolean `true`:

![immagine](https://github.com/user-attachments/assets/d0d4f280-e083-45de-aeac-e300d9de1947)